universal-application-tool-0.0.1/test/forms/TextQuestionFormTest.java

```diff
@@ -24,16 +25,18 @@ public void getBuilder_returnsCompleteBuilder() throws Exception {
     form.setTextMaxLength(6);
     QuestionDefinitionBuilder builder = form.getBuilder();
 
-    // The QuestionForm does not set version, which is needed in order to build the
-    // QuestionDefinition. How we get this value hasn't been determined.
+    // The QuestionForm does not set version or lifecycle stage.
+    // A first question is a draft in version 1 - set those here.
     builder.setVersion(1L);
+    builder.setLifecycleStage(LifecycleStage.DRAFT);
 
     TextQuestionDefinition expected =
         new TextQuestionDefinition(
             1L,
             "name",
             Path.create("my.question.path.name"),
             "description",
+            LifecycleStage.ACTIVE,
```

Comment:
Thoughts (blocking): it is very confusing to me that a DRAFT question is equivalent to an ACTIVE question. Would you add comments both here and in `QuestionDefinition#equals` why that is the case?

Reply:
Ah, I didn't see that question definition overrode equals().  I thought it was an autovalue.  Added this to the equals and fixed new breaking tests.